cpp/proto/serializer.cc

```diff
@@ -106,6 +107,9 @@ SerializeResult Serializer::SerializeV1STHSignatureInput(
 }
 
 
+////////////////////////////////////////////////////////////////////////////////
+//                                SERIALIZATION //
```

Comment:
If you're going to make funny boxes like that, you should do them properly? :wink: 

(the `//` on the right side should go further? I don't really care much for those kinds of markers, makes me think things should be in a different file when I see them)